### DISCUSSION TITLE
Questions about Github Discussions

### DISCUSSION THREAD

**Discussion by Pertex** [textadventures/textadventures.co.uk#8] [General]
@alexwarren thanx for the invitation. I am delighted to be able to help with the moderation. That's why I've created this discussion as a test with a few questions

Can I break something by accident or stupidity as a moderator? If not, do we habe the option to appoint more moderators if necessary? Do you know if there are active moderators in the old forum ?

Is it possible to create a discussion that can only be read by the moderators?

It is already apparent that not much will take place directly under textadventure. I would therefore suggest leaving it as it is and moving discussions from there to Quest or Squiffy. Perhaps textadventures could be reserved for general content from Alex such as announcements, future plans, (the old blog so to speak).

It is possible to move discussions, isn't it?

I would like to pin one discussion each under Quest and Squiffy, briefly showing the basics of the discussions and the Mermaid graph. Is that OK?

**Comment by alexwarren** [comment]
Thanks Pertex! I think we'll be figuring things out and evolving as we go along, but to answer what I do know:

There's documentation on moderation [here](https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions). I've given you "triage" permissions (I think only on the Quest repo so far? But I can add you on the others) which lets you do moderation but it doesn't grant any other admin powers, so it doesn't look like there's much you can break accidentally, and we can give the same permissions to others too. There are a handful of existing moderators on the textadventures.co.uk site.

I don't think there's a way of creating a mod-only discussion. I'd suggest Discord is the best place for that right now as we have a private channel there.

Discussions can be transferred between repos (or at least they can from this parent textadventures one). I think you're right that this one will be the quietest, and we'll keep it for discussions about the site in general.

Happy for you to pin anything you see fit.

**Comment by KVonGit** [comment]
> Do you know if there are active moderators in the old forum ?

IFforClassroom is in the #moderators channel on Discord, and XanMag will likely be showing up after his busy season lets up.[^1]

I think Silver is also a mod. I used to see a post every few months, but I haven't noticed any recently. (That might change when Silver sees Alex is back?[^2])

[^1]: XM still has to write part 3 of his trilogy. He'll be back, now that things are working properly and almost everyone else is back.
[^2]: The only interactions Silver and I ever had involved berating poorly written games.
